### ISSUE TITLE
Problem with the database and the file language (FR)

### ISSUE THREAD

**Issue by patatrac** [diddipoeler/joomleague-2-komplettpaket#17]
Hello, 
I installed the package yesterday and I encounter error messages regarding the database type:
joomleague_prediction_game' doesn't exist SQL=SELECT pre.*, u.name AS editor FROM elcc_joomleague_prediction_game AS pre LEFT JOIN elcc_users AS u ON u.id = pre.checked_out ORDER BY pre.name LIMIT 0, 20
And the language file does not seem to take into account, I have all the terms that appear in this way:
COM_JOOMLEAGUE_JL_ADMIN_PGAMES_NAME

**Comment by svdoldie** [comment]
hi,

you have to do pred_install_db_tables.php
in jl-backend -> administration -> updates

>  -------- Original-Nachricht --------
>  Datum: Mon, 11 Mar 2013 05:33:02 -0700
>  Von: patatrac notifications@github.com
>  An: diddipoeler/joomleague-2-komplettpaket joomleague-2-komplettpaket@noreply.github.com
>  Betreff: [joomleague-2-komplettpaket] Problem with the database and the file language (FR) (#17)
>

**Comment by patatrac** [comment]
I tried, but it does not work ...

**Comment by svdoldie** [comment]
hmm, and this jl_update_db_tables.php ?

**Comment by patatrac** [comment]
Not better ...
I had version 2.0.63 and installed before installing the package. Does the fact that I did not uninstall the previous version first could be causing my problem?

**Comment by svdoldie** [comment]
yes, you have to uninstall jl-original, and then install diddis version.
then do the update.php und the pred-db-install.php

**Comment by patatrac** [comment]
aaah!
I tried on my side and it worked :)
I still have my problem with the language file

**Comment by svdoldie** [comment]
the language file is not ready yet

**Comment by patatrac** [comment]
ok

**Comment by patatrac** [comment]
I tried to do the mods on my side in this case.
Thank you for your help.

**Comment by patatrac** [comment]
I made an attempt to create directly with this version:
- Unable to create a league, the button to save does not work.
- Unable to create automatically a league or a season since the inception of the project.
- I can install another version (2.50) to correctly create my championship.

**Comment by patatrac** [comment]
In fact, the record button is almost never works (creating league, new club, etc ...)

**Comment by svdoldie** [comment]
try to save global settings

**Comment by patatrac** [comment]
Can you tell me where do? I can not find where it is

**Comment by patatrac** [comment]
I found and it works
Thanks

**Comment by patatrac** [comment]
Is this extension is currently in production on a site?

**Comment by svdoldie** [comment]
i do not understand the question?

**Comment by patatrac** [comment]
Sorry, I'm using google Traduct I rephrase my sentence.

**Comment by patatrac** [comment]
Is anyone using this extension on a website. I wanted to see the result, because I can not get out to the setting. Maybe it could help me ...

**Comment by patatrac** [comment]
Unless there is a user guide?

**Comment by svdoldie** [comment]
diddis site http://fussballineuropa.de/index.php?option=com_content&view=featured&Itemid=490 is using the komponent, or do you mean the predictiongame-extension? 
there is an wiki, but it's just at the beginning

**Comment by patatrac** [comment]
Yes, I mean the prediction extension

**Comment by svdoldie** [comment]
i know diddi's testsite only http://joomla25.diddipoeler.de/index.php/de/?Itemid=961

**Comment by patatrac** [comment]
I bother you again, would you know how to get values ​​in dropdown list for the field : JL_XML_PREDICTIONENTRY_STATE_PARAM_LABEL
on the basic options menu link type : JL_XML_PREDICTIONENTRY_LAYOUT_TITLE 
Mine is empty

**Comment by svdoldie** [comment]
you have to create a predictiongame first

**Comment by patatrac** [comment]
Yes I do

**Comment by patatrac** [comment]
And I related to my project

**Comment by svdoldie** [comment]
it's all plublished too?

**Comment by patatrac** [comment]
shame on me ...

**Comment by svdoldie** [comment]
:D

**Comment by patatrac** [comment]
It still works much better now :)
For cons, I want to make a menu link to the next day predictions, and I always fall on the first day, without being able to change the day.

**Comment by svdoldie** [comment]
is gameday in the projectsettings on "manual"?

**Comment by patatrac** [comment]
I found an option that could fit in the general settings but the backup does not retain the setting, but I found how to change the day for the time manually, but this is not practical.

**Comment by patatrac** [comment]
It is not easy to understand the settings to change without French language file.
Thank you for your help ;)

**Comment by patatrac** [comment]
Hello,
Do you know which file I can change the appearance of the table for prediction entry page?
